README.md

```diff
@@ -79,7 +79,9 @@ So far we have only two user roles: participants and group organizers
 
 - I can add a food sponsor to the event with a link to their website as a way of thanking them for food.
 
-I can see how many times a participant has come to the event as well as their attendance rate
+- I can see how many times a participant has come to the event as well as their attendance rate
```

Comment:
Not really sure this is something we need for the MVP. Let's raise it in Discord 😄

Reply:
@francocorreasosa Sorry this was merged  I just added for matting to it.

Reply:
That's my changes silly me forgot about the bulleting

Anyway it's a good features that we need for MVP, the current schema also has already adjusted with this story